modules/RTC/DataChannels.js

```diff
@@ -197,7 +197,13 @@ DataChannels.prototype._onXXXEndpointChanged = function (xxx, userResource) {
             jsonObject.colibriClass = (upper + 'EndpointChangedEvent');
             jsonObject[lower + "Endpoint"]
                 = (userResource ? userResource : null);
-            dataChannel.send(JSON.stringify(jsonObject));
+            //Tries to fix errors on peer connection close.
+            try {
+                dataChannel.send(JSON.stringify(jsonObject));
+            } catch (e) {
+                // FIXME: Handle properly
```

Comment:
How would the actual fix look like?
I am trying to understand why are we delaying the actual implementation with a temporary hack.